pkgs/development/tools/racerd/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper }:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  version = "0.1.1";
+  name = "racerd-${version}";
+
+  src = fetchFromGitHub {
+    owner = "jwilm";
+    repo = "racerd";
+    rev = "9b63607132ab7f9aa32eb85eb45707c1b8f70990";
+    sha256 = "0kggikin8hyq71hwbapczyvik39wwycvgdxigzhgivlcpss2wmdx";
+  };
+
+  depsSha256 = "";
```

Comment:
@matthiasbeyer Sorry, this is not documented (yet) nor there are tools to calculate it.

At the moment, I think the only way to get it is for you to put a properly-formatted but bogus hash there (make sure it's a hash that is not used in other artifacts), then try to build it and wait for Nix to tell you that there is a hash mismatch. From this error message, you can extract the correct hash.

Usually, I just copy/paste some other hash (like the sha256 hash right above it) and then change the last character to a zero, then do the above procedure to find out the correct hash.